packs/equipment/bewitching-bloom-purple-iris.json

```
@@ -1,12 +1,12 @@
 {
     "_id": "2Xpjq6eYMqvBL6eg",
-    "img": "systems/pf2e/icons/default-icons/equipment.svg",
+    "img": "icons/magic/symbols/arrowhead-green.webp",
     "name": "Bewitching Bloom (Purple Iris)",
     "system": {
         "baseItem": null,
         "containerId": null,
         "description": {
-            "value": "<p>While dormant, this tattoo appears to be a simple flower bud, but when activated the flower swiftly blossoms, remaining that way until the next time you make your daily preparations. These blooms are colorful, elegant representations of purple iris flowers.</p>\n<p><strong>Activate</strong><span class=\"action-glyph\">2</span> envision</p>\n<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p><strong>Effect</strong> Choose a willing ally you can see within 30 feet. Your ally takes on a regal bearing, exuding the presence of confident royalty. The bloom casts a 5th-level <em>@UUID[Compendium.pf2e.spells-srd.Item.Command]</em>spell (DC 28) selecting targets within range of the ally. Each target that fails the save must fall @UUID[Compendium.pf2e.conditionitems.Item.Prone] and pays homage to your ally.</p>"
+            "value": "<p>While dormant, this tattoo appears to be a simple flower bud, but when activated the flower swiftly blossoms, remaining that way until the next time you make your daily preparations. These blooms are colorful, elegant representations of purple iris flowers.</p>\n<p><strong>Activate</strong><span class=\"action-glyph\">2</span> envision</p>\n<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p><strong>Effect</strong> Choose a willing ally you can see within 30 feet. Your ally takes on a regal bearing, exuding the presence of confident royalty. The bloom casts a 5th-level <em>@UUID[Compendium.pf2e.spells-srd.Item.Command]</em> spell (DC 28) selecting targets within range of the ally. Each target that fails the save must fall @UUID[Compendium.pf2e.conditionitems.Item.Prone] and pays homage to your ally.</p>"
```

```suggestion
            "value": "<p>While dormant, this tattoo appears to be a simple flower bud, but when activated the flower swiftly blossoms, remaining that way until the next time you make your daily preparations. These blooms are colorful, elegant representations of purple iris flowers.</p>\n<p><strong>Activate</strong><span class=\"action-glyph\">2</span> envision</p>\n<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p><strong>Effect</strong> Choose a willing ally you can see within 30 feet. Your ally takes on a regal bearing, exuding the presence of confident royalty. The bloom casts a 5th-level @UUID[Compendium.pf2e.spells-srd.Item.Command] spell (DC 28) selecting targets within range of the ally. Each target that fails the save must fall @UUID[Compendium.pf2e.conditionitems.Item.Prone] and pays homage to your ally.</p>"
```

It's emphasized by CSS now, no need for the tags. Post player core someone can run something to remove all the existing tags to save however much space in compendiums.